05/app.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { createRoot } from "react-dom/client";
+
+const root = createRoot(document.querySelector("#root"));
+
+class Weather extends React.Component {
+  key = "8166bd1b540b42039917fe1e6e4d13f0";
+  url = `https://api.weatherbit.io/v2.0/current?key=${this.key}&lang=pl`;
+
+  constructor(props) {
+    super(props);
+
+    const { lat, lng } = this.props;
+    this.state = {
+      data: null,
+      lat,
+      lng,
+    };
+  }
+
+  async getWeather() {
```

Comment:
Jeszcze lepszym rozwiązaniem byłoby podzielenie tego komponentu w taki sposób, że odpytanie do api to jest w osobnym pliku, a tutaj po prostu ustawiamy stan :)